docs/excel/custom-functions-json-autogeneration.md

```diff
@@ -52,21 +52,33 @@ There should also be a call to `CustomFunctions.associate("id", functionName);`
 
 #### id
 
-The id is used as the invariant identifier for the custom function stored in the document. It should not change.
+The `id` is used as the invariant identifier for the custom function stored in the document. It should not change.
```

Comment:
```suggestion
The `id` is an invariant identifier for the custom function.
```
Not sure what "stored in the document" means. What document?
 "it should not change" is redundant.